bin/main.rb

```diff
@@ -1,2 +1,45 @@
 #!/usr/bin/env ruby
-p 'hello, world!'
+
+require_relative '../lib/game_player'
+require_relative '../lib/game_board'
+
+def display_instructions
+  puts 'This is a tic-tac-toe game'
+  puts 'Press the Enter key to start the game'
+  puts 'Player 1 is "X" and Player 2 is "O"'
+  puts
```

Comment:
- [ ] Remove puts

Reply:
Those puts are in the main file, Can you please explain why should I remove them?